application/displayblock.class.inc.php

```diff
@@ -1677,11 +1677,11 @@ public function GetRenderContent(WebPage $oPage, $aExtraParams = array(), $sId)
 						{
 							if (array_key_exists('down', $aRelationInfo))
 							{
-								$aActions[$sRelationCode.'_down'] = array ('label' => $aRelationInfo['down'], 'url' => "{$sRootUrl}pages/$sUIPage?operation=swf_navigator&relation=$sRelationCode&direction=down&class=$sClass&id=$id{$sContext}") + $aActionParams;
+								$aActions[$sRelationCode.'_down'] = array ('label' => $aRelationInfo['down'], 'url' => "{$sRootUrl}pages/$sUIPage?operation=view-relations&relation=$sRelationCode&direction=down&class=$sClass&id=$id{$sContext}") + $aActionParams;
```

Comment:
I know @rquetiez told you to put a dash but it was a typo, we rather use underscores for operations like in the rest of the PHP endpoints.

```suggestion
								$aActions[$sRelationCode.'_down'] = array ('label' => $aRelationInfo['down'], 'url' => "{$sRootUrl}pages/$sUIPage?operation=view_relations&relation=$sRelationCode&direction=down&class=$sClass&id=$id{$sContext}") + $aActionParams;
```

Reply:
Thanks, will combine them in 1 commit if no problem..

Reply:
Yeah sure 😊